### PR TITLE
Fix analyze warnings for imports

### PR DESCRIPTION
**Pull request by mpozy**

### CODE REVIEW

File: lib/src/mapper/mapper.dart [hunk 1]
```diff
@@ -1,11 +1,4 @@
 import 'package:dolbyio_comms_sdk_flutter/dolbyio_comms_sdk_flutter.dart';
```
Comment (graduad) [inline lib/src/mapper/mapper.dart:1]: This should not be imported anywhere in the SDK.

File: lib/src/mapper/mapper.dart [hunk 2]
```diff
@@ -1,11 +1,4 @@
...
-import 'package:dolbyio_comms_sdk_flutter/src/sdk_api/models/conference.dart';
-import 'package:dolbyio_comms_sdk_flutter/src/sdk_api/models/events.dart';
-import 'package:dolbyio_comms_sdk_flutter/src/sdk_api/models/participant.dart';
-import 'package:dolbyio_comms_sdk_flutter/src/sdk_api/models/video_presentation.dart';
-
-import '../sdk_api/models/recording.dart';
-import '../sdk_api/models/streams.dart';
```
Comment (graduad) [inline lib/src/mapper/mapper.dart:8]: As you will probably need this please investigate what is the preferred style in Dart,
* using absolute imports, ie. `import 'package:<path>.'`, or
* using relative imports ie. `import '../<path>'`.
There after please correct this as appropriate.

File: lib/src/sdk_api/command_service.dart [hunk 3]
```diff
@@ -3,7 +3,6 @@ import 'dart:async';
 import 'package:dolbyio_comms_sdk_flutter/dolbyio_comms_sdk_flutter.dart';
```
Comment (graduad) [inline lib/src/sdk_api/command_service.dart:3]: This should not be imported anywhere in the SDK.

File: lib/src/sdk_api/models/events.dart [hunk 4]
```diff
@@ -1,7 +1,5 @@
 import 'package:dolbyio_comms_sdk_flutter/dolbyio_comms_sdk_flutter.dart';
```
Comment (graduad) [inline lib/src/sdk_api/models/events.dart:1]: This should not be imported anywhere in the SDK.

File: lib/src/sdk_api/notification_service.dart [hunk 5]
```diff
@@ -5,9 +5,6 @@ import 'package:dolbyio_comms_sdk_flutter/src/dolbyio_comms_sdk_native_events.da
 
 import '../../dolbyio_comms_sdk_flutter.dart';
```
Comment (graduad) [inline lib/src/sdk_api/notification_service.dart:6]: This should not be imported anywhere in the SDK.

File: test/service/dolbyio_comms_conference_service_test.dart [hunk 6]
```diff
@@ -1,6 +1,5 @@
 import 'package:dolbyio_comms_sdk_flutter/dolbyio_comms_sdk_flutter.dart';
```
Comment (graduad) [inline test/service/dolbyio_comms_conference_service_test.dart:1]: This should not be imported anywhere in the SDK.

File: test/service/dolbyio_comms_file_presentation_service_test.dart [hunk 7]
```diff
@@ -1,5 +1,4 @@
 import 'package:dolbyio_comms_sdk_flutter/dolbyio_comms_sdk_flutter.dart';
```
Comment (graduad) [inline test/service/dolbyio_comms_file_presentation_service_test.dart:1]: This should not be imported anywhere in the SDK.

File: test/service/dolbyio_comms_video_presentation_service_test.dart [hunk 8]
```diff
@@ -1,8 +1,5 @@
 import 'package:dolbyio_comms_sdk_flutter/dolbyio_comms_sdk_flutter.dart';
```
Comment (graduad) [inline test/service/dolbyio_comms_video_presentation_service_test.dart:1]: This should not be imported anywhere in the SDK.

File: test/test_helpers.dart [hunk 9]
```diff
@@ -1,6 +1,4 @@
 import 'package:dolbyio_comms_sdk_flutter/dolbyio_comms_sdk_flutter.dart';
```
Comment (graduad) [inline test/test_helpers.dart:1]: This should not be imported anywhere in the SDK.

File: test_app/lib/screens/participant_screen/participant_screen.dart [hunk 10]
```diff
@@ -1,6 +1,5 @@
 import 'package:flutter/material.dart';
 import 'dart:async';
-import 'dart:developer' as developer;
 import 'package:dolbyio_comms_sdk_flutter/dolbyio_comms_sdk_flutter.dart';
```
Comment (graduad) [inline test_app/lib/screens/participant_screen/participant_screen.dart:3]: This should not be imported anywhere in the SDK.

**Review by graduad — CHANGES_REQUESTED** [review]

File: test/service/dolbyio_comms_file_presentation_service_test.dart (hunk 7 above)
Comment (mpozy) [reply test/service/dolbyio_comms_file_presentation_service_test.dart:1]: So we should change the test.

**Review by graduad — DISMISSED** [review]

**Review by graduad — APPROVED** [review]

**Pull request merged**